zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Root.java

```diff
@@ -194,6 +187,18 @@ private static void loadExistingCollectionsIntoScheduler() {
         }
     }
 
+    static void alertOnInProgressCollections(Collections.CollectionList collections, Notifier notifier) {
+        info().log("zebedee root: checking existing collections for in progress approvals");
+        collections.stream()
+                .filter(c -> c.getDescription().approvalStatus == ApprovalStatus.IN_PROGRESS
+                        || c.getDescription().approvalStatus == ApprovalStatus.ERROR)
+                .forEach(c -> {
+                    info().data("collectionId", c.getDescription().getId())
+                            .data("type", c.getDescription().getType().name())
+                            .log("zebedee root: collection approval is in error or in progress state on zebedee startup");
+                    notifier.collectionAlarm(c, "Collection approval is in IN_PROGRESS or ERROR state on zebedee startup. It may need to be re-approved manually.");
```

Comment:
You could probably make the message a const as its the same message each time. Plus keeps like line short 😁

Reply:
I debated doing this, but decided not to as the slack message has additional formatting and the suggested fix. As I understand the log message should be all lower case without formatting.